app/javascript/dashboard/routes/dashboard/settings/personal/HotKeyCard.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+  <div
+    class="flex flex-col gap-4 w-full h-fit sm:max-h-[220px] p-4 sm:max-w-[350px] rounded-md border border-solid border-ash-200"
+    :class="{
+      'border border-solid border-primary-300 ': active,
```

Comment:
@muhsin-k We do not need to add the `border border-solid` here again already added at the top. And I think we don't need to add the `border-solid` because the default value for the border will be the same.